tests/sysauth/sssd.pm

```diff
@@ -29,10 +29,7 @@ sub run {
     if (is_sle) {
         assert_script_run 'source /etc/os-release';
         if (is_sle '>=15') {
-            if (script_run('SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300) != 0) {
-                record_soft_failure 'bsc#1124318 - Fail to get PackageHub Pool Metadata - running the command again as a workaround';
-                assert_script_run 'SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300;
-            }
+            add_suseconnect_product('PackageHub');
```

Comment:
Better only fix for sssd, and also if you find other modules where PackageHub is registered.

```suggestion
            my $version = get_var('VERSION');
            my $arch = get_var('ARCH');
            my $params = '';
            my $timeout = 300;
            $retry = 1;
            add_suseconnect_product('PackageHub', $version, $arch, $params, $timeout, $retry);
```

This is just the suggestion, you may need to tweak the values, like the version `15-SP1 -> 15.1`.

Reply:
is_sle '>=15' is okay here and it covers both I think. Will check this change and provide new verification tests, thanks

Reply:
I mean the version you send to the function: `my $version = get_var('VERSION')`

Reply:
use my $version //= '${VERSION_ID}';